Gruntfile.js

```diff
@@ -144,7 +101,7 @@ module.exports = function(grunt) {
           }
         },
         files: {
-          'dist/umd/qrious.min.js': 'src/runtime/browser.js'
+          'dist/qrious.min.js': 'src/runtime/browser.js'
```

Comment:
Since UMD is now the only distribution format that's generated by the build, I've moved its files to the `dist` directory directly instead of under the `umd` sub-directory.

This will be a breaking changed for those using the CDN download links referenced in the README, but I fear that it's best. If I get enough kickback, I might add symbolic links, otherwise, I will recommend that they simply change their URLs as follows for the 2.3.0:

* https://cdn.rawgit.com/neocotic/qrious/2.3.x/dist/umd/qrious.js
* https://cdn.rawgit.com/neocotic/qrious/2.3.x/dist/umd/qrious.min.js

The download links in the README have been updated as part as this PR.